frontend/src/pages/voting/index.tsx

```diff
@@ -208,30 +214,52 @@ export const VotingPage = () => {
         </Box>
       </Box>
 
-      {loggedIn && (loading || cannotVoteErrors || error) && (
-        <Box
-          sx={{
-            maxWidth: "container",
-            mx: "auto",
-            px: 3,
-          }}
-        >
-          {!cannotVoteErrors && error && (
-            <Alert variant="alert">{error.message}</Alert>
-          )}
-
-          {cannotVoteErrors && error && (
-            <Alert variant="alert">
-              <Link path="/tickets">
-                <FormattedMessage id="voting.buyTicketToVote" />
-              </Link>
-            </Alert>
-          )}
-          {loading && (
-            <Alert variant="info">
-              <FormattedMessage id="voting.loading" />
-            </Alert>
-          )}
+      {userCannotVote && (
+        <Box>
+          <Box
+            sx={{
+              maxWidth: "container",
+              mx: "auto",
+              px: 3,
+              py: 0,
+            }}
+          >
+            {!cannotVoteErrors && error && (
+              <Alert variant="alert">{error.message}</Alert>
+            )}
+
+            {cannotVoteErrors && error && (
+              <Box sx={{ pt: 4 }}>
+                <Heading>
+                  <FormattedMessage id="voting.errors.cannotVote.heading" />
+                </Heading>
+
+                <Text my={4}>
+                  <FormattedMessage
+                    id="voting.errors.cannotVote.body"
+                    values={{
+                      linkVotingInfo: (
+                        <Link path="/voting-info">
+                          <FormattedMessage id="voting.errors.cannotVote.linkVotingInfo.text" />
+                        </Link>
+                      ),
+                      linkTicket: (
+                        <Link path="/tickets">
+                          {" "}
```

Comment:
this {" "} space looks terrible. Can we add it to the string in the locale?

```suggestion
```

Reply:
Was a mistake